docs/DeveloperGuide.adoc

```diff
@@ -767,6 +785,37 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Email Intents
+
+*MSS*
+
+1. User clicks on email button.
+2. AddressBook opens up the Gmail webpage.
```

Comment:
What if the user hasn't registered his/her Gmail account? Is Gmail is only email service allowed here?